api_access/views.py

```
@@ -485,20 +499,42 @@ def post(self, request):
+                transaction.on_commit(
+                    lambda workflow_audit=auditor.audit, request_id=auditor.workflow.request_id: async_task(
+                        notify_for_audit,
+                        workflow_audit=workflow_audit,
+                        timeout=60,
+                        task_name=f"permission-request-{request_id}",
+                    )
+                )
         except AuditException:
             raise serializers.ValidationError(
                 {"errors": "Failed to create approval flow, please contact admin."}
             )
```

Ruff (0.15.11)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)